ports/servoshell/desktop/app.rs

```diff
@@ -435,6 +509,9 @@ impl ApplicationHandler<WakerEvent> for App {
         // Consume and handle any events from the Minibrowser.
         self.handle_servoshell_ui_events();
```

Comment:
I suggest we can disable this for webdriver test.
```suggestion
    if self.opts.webdriver_port.is_none(){
        self.handle_servoshell_ui_events();
    }
```